openmc/model/surface_composite.py

```diff
@@ -1,10 +1,11 @@
 from abc import ABC, abstractmethod
 from copy import copy
+from math import sqrt
+from numpy import array
```

Comment:
For consistency with the rest of the codebase:
```suggestion

import numpy as np
```